drivers/src/error.rs

```diff
@@ -53,6 +53,9 @@ pub enum CaliptraComponent {
 
     /// CSRNG
     Csrng = 13,
+
+    /// Runtime firmware
+    Runtime = 14,
```

Comment:
I don't think this is where @vsonims intended the runtime errors to go. Maybe put a TODO here saying something like "move this somewhere more idiomatic".

@rusty1968 has plans to revamp how the errors are organized.

Reply:
Done (once https://github.com/chipsalliance/caliptra-sw/pull/220 is merged I'll switch to use that mechanism)